role-assignment.tf

```diff
@@ -0,0 +1,7 @@
+resource "azurerm_role_assignment" "github_runner" {
```

Comment:
`github_runner` doesn't seem right.

if there's no better name and it's the only one of this resource you can just use `this`

Reply:
sorry copy and paste issue :P